test_caseinsensitivedict.py

```diff
@@ -100,6 +110,13 @@ def test_keys(self):
         # popitem()
         #     Remove and return a (key, value) pair from the dictionary.  Pairs
         #     are returned in LIFO order.
+    def test_popitem(self):
```

Comment:
FYI these methods are not working, so shouldn't be merged to the master branch, until they're done.

Reply:
if these are working now, say so and resolve this comment